scripts/core/ui-utils.tsx

```diff
@@ -2,11 +2,23 @@ import {showModal} from '@superdesk/common';
 import {Modal} from 'superdesk-ui-framework/react';
 import React from 'react';
 
+type modalPosition =
+'center'
+| 'top'
+| 'bottom'
+| 'left'
+| 'right'
+| 'top-left'
+| 'top-right'
+| 'bottom-left'
+| 'bottom-right';
+
 export const ui = {
-    alert: (message: string) => (
+    alert: (message: string, position: modalPosition = 'top') => (
         showModal(({closeModal}) => {
             return (
                 <Modal
+                    position={position}
```

Comment:
it'd be better to simply hardcode this to top. An alert is supposed to be one/two sentences, so no other positioning would be desired